### PR TITLE
pyrit: add page

### PR DESCRIPTION
**Pull request by TerribleCodes**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented: 0.5.0**
#6863

### CODE REVIEW

File: pages/linux/pyrit.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`pyrit -e {{ESSID}} create_essid`
+
+- Read and analyze packet capture file:
+
+`pyrit -r {{path/to/file}} analyze`
+
+- Read and import passwords to the database:
+
+`pyrit -i {{path/to/file}} import_passwords`
```
Comment (TerribleCodes) [inline pages/linux/pyrit.md:24]: Can use `import_unique_passwords` or `unique_passwords` here. The difference is `import_unique_passwords`is faster than the `import_passwords` command. Need help here too.

File: pages/linux/pyrit.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`pyrit list_cores`
+
+- Set ESSID:
+
+`pyrit -e {{ESSID}} create_essid`
+
+- Read and analyze packet capture file:
+
+`pyrit -r {{path/to/file}} analyze`
```
Comment (TerribleCodes) [inline pages/linux/pyrit.md:20]: File extensions for capture files are `.cap` and `.pcap`. Need help here.

File: pages/linux/pyrit.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`pyrit benchmark`
+
+- List available cores:
+
+`pyrit list_cores`
+
+- Set ESSID:
+
+`pyrit -e {{ESSID}} create_essid`
```
Comment (EmilySeville7cfg) [inline pages/linux/pyrit.md:16]:
```suggestion
- Set [e]SSID:

`pyrit -e "{{ESSID}}" create_essid`
```

Optional fix: mnemonic can be added.
Optional fix: quoting can be added if the placeholder can contain special globbing or other characters to POSIX-compatible shell.

File: pages/linux/pyrit.md (hunk 2 above)
Comment (EmilySeville7cfg) [inline pages/linux/pyrit.md:20]:
```suggestion
- [r]ead and analyze a specific packet capture file:

`pyrit -r {{path/to/file.cap|path/to/file.pcap}} analyze`
```

Optional fix: mnemonic can be added.
Optional fix: two extensions can be presented via `|` syntax.

File: pages/linux/pyrit.md (hunk 1 above)
Comment (EmilySeville7cfg) [inline pages/linux/pyrit.md:24]:
```suggestion
- Read and [i]mport passwords to the current database:

`pyrit -i {{path/to/file}} {{import_unique_passwords|unique_passwords|import_passwords}}`
```

Almost the same issue as above.
Optional fix: as database placeholder is not found, it's assumed that this example referred to the current database.

File: pages/linux/pyrit.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`pyrit -r {{path/to/file}} analyze`
+
+- Read and import passwords to the database:
+
+`pyrit -i {{path/to/file}} import_passwords`
+
+- Export passwords from database to a file:
+
+`pyrit -o {{path/to/file}} export_passwords`
```
Comment (EmilySeville7cfg) [inline pages/linux/pyrit.md:28]:
```suggestion
- Exp[o]rt passwords from database to a specific file:

`pyrit -o {{path/to/file}} export_passwords`
```

Almost the same issue as above.
Optional fix: `specific` may sound better.

File: pages/linux/pyrit.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Translate passwords with Pired Master Keys:
+
+`pyrit batch`
+
+- Read the capture file and crack the password:
+
+`pyrit -r {{path/to/file}} attack_db`
```
Comment (EmilySeville7cfg) [inline pages/linux/pyrit.md:36]:
```suggestion
- [r]ead the capture file and crack the password:

`pyrit -r {{path/to/file}} attack_db`
```

Optional fix: mnemonic can be added.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🚀

**Review by kbdharun — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]
No fixes found, approved.
Thanks for a contribution! 🚀

**Pull request merged**